docs/includes/v3_backup_steps_common.rst

```diff
@@ -0,0 +1,146 @@
+      cp $SD_OLD/tor_v3_keys.json $SD_NEW/tor_v3_keys.json
+      cp $SD_OLD/sd-backup-old.tar.gz $SD_NEW/sd-backup-old.tar.gz
+
+   You will also need to copy the old instance's *Submission Public Key*,
+   *Ossec Alert Public Key*, and, if configured, the *Journalist Alert Public Key*.
+   Assuming that these are named ``SecureDrop.asc``, ``ossec.asc``, and
+   ``journalist.asc`` respectively, run the following commands:
+
+   .. code:: sh
+
+      cp $SD_OLD/SecureDrop.asc $SD_NEW/SecureDrop.asc
+      cp $SD_OLD/ossec.asc $SD_NEW/ansible-base/ossec.asc
+      cp $SD_OLD/journalist.asc $SD_NEW/ansible-base/journalist.asc
+
+#. *(Optional):* If your old instance was configured to provide the *Source
+   Interface* via HTTPS, you should also copy across the certificate, certificate
+   key, and chain file. Assuming that these are named ``sd.crt``, ``sd.key``, and
+   ``ca.crt`` respectively, run the following commands:
+
+   .. code:: sh
+
+      cp $SD_OLD/sd.{crt,key} $SD_NEW/
+      cp $SD_OLD/ca.crt $SD_NEW/
+
+#. Install Ubuntu 20.04 on the *Application* and *Monitor Servers*, following
+   the :doc:`server setup instructions<../servers>` to install with the correct
+   settings, test connectivity, and set up SSH keys to allow for
+   *Admin Workstation* access.
+
+#. Reinstall SecureDrop on the servers, following the :doc:`installation
+   instructions <../install>`. During the configuration stage
+   (``./securedrop-admin sdconfig``), the values will be prepopulated based on
+   the old instance's configuration. Press **Enter** to accept each value,
+   except when you are asked if you want to enable v2 onion services - instead,
+   type ``no``.
+
+   Proceed through the installation, finishing with
+   ``./securedrop-admin tailsconfig``. If SSH-over-Tor is configured, run
+   ``ssh app uptime`` and ``ssh mon uptime``  in the Terminal to verify SSH
+   connectivity and add the new onion URLs to your ``known_hosts`` file.
+
+#. Restore from the old instance's backup (e.g. ``sd-backup-old.tar.gz``) using
+   the Terminal command:
+
+   .. code:: sh
+
+       ./securedrop-admin restore sd-backup-old.tar.gz
+
+   The restore task will proceed for some time, removing v2 services if a v2+v3
+   backup was used, and then will fail with the message:
+
+   .. code-block:: none
+
+     ssh_exchange_identification: Connection closed by remote host
+
+   during the ``Wait for Tor to reload`` task. This is expected; the
+   *Application Server*'s SSH onion service address was updated to the old
+   instance's address during the restore process, leaving it temporarily
+   unreachable.
+
+#. Copy the old instance's v3 onion service details into place on the
+   *Admin Workstation* and repair SSH access using the Terminal commands:
+
+   .. code:: sh
+
+      cd ~/Persistent/securedrop
+      cp $SD_OLD/app-{journalist,ssh}.auth_private $SD_NEW/
+      cp $SD_OLD/app-sourcev3-ths $SD_NEW
```

Comment:
Trailing slash missing here (both will work just fine; the trailing slash is nice to ensure it errors out in case the directory is not present as expected).

Reply:
fixed in 2e6a9b3